middleware/logging.go

```diff
@@ -39,7 +41,18 @@ func (l Log) Wrap(next http.Handler) http.Handler {
 		var buf bytes.Buffer
 		wrapped := newBadResponseLoggingWriter(w, &buf)
 		next.ServeHTTP(wrapped, r)
-		statusCode := wrapped.statusCode
+
+		statusCode, writeErr := wrapped.statusCode, wrapped.writeError
+
+		if writeErr != nil {
+			if errors.Is(writeErr, context.Canceled) {
```

Comment:
I think this forces consumers to upgrade to Go 1.13, which should be noted

Reply:
Hrmm, yeah. But where should I note it though?

Reply:
In the PR description.
Be nice if it was self-contained rather than making people read something else.